overlord/assertstate/assertstate.go

```diff
@@ -726,6 +726,47 @@ func ValidationSetAssertionForEnforce(st *state.State, accountID, name string, s
 	return vs, err
 }
 
+func EnforceValidationSet(st *state.State, accountID, name string, sequence, userID int, snaps []*snapasserts.InstalledSnap, ignoreValidation map[string]bool) error {
```

Comment:
Since those are exported, a tiny doc comment here and for MonitorValidationSet would be useful